### ISSUE TITLE
New Grafana dashboard to capture pg_stat_statements data

### ISSUE THREAD

**Issue by Red-HAP** [project-koku/koku#2102]
## User Story

As a developer, I want to be able to see the query performance in a monitor dashboard.

Add a dashboard to Grafana to live-query pg_stat_statements in production. Allow for a 10-second query poll by default, shorter times can be added, if needed.

The widget in this dashboard can be modelled on the work done for dev-grafana

## Impacts

Grafana

## Role

Developer

## Assumptions

- [ ] A new dashboard can be created in production Grafana

## API Details

API unaffected

## UI Details

UI unaffected

## Acceptance Criteria

- [ ] New grafana dashblard can be created to live-query db every 10 seconds by default.

**Comment by Red-HAP** [comment]
[pg_stat_statements_grafana_widget.json.gz](https://github.com/project-koku/koku/files/4552252/pg_stat_statements_grafana_widget.json.gz)

The above file contains the widget definition from dev grafana changing the call to the get_pg_stat_statements() function vs the table.

**Comment by Red-HAP** [comment]
The following was added to the production `koku-app` config map data:
```
  query-yaml: |-
    pg_stat_statements:
      query: "SELECT query, calls, total_time, min_time, max_time, mean_time, stddev_time, rows, shared_blks_hit, shared_blks_read, shared_blks_dirtied, shared_blks_written, local_blks_hit, local_blks_read, local_blks_dirtied, local_blks_written, temp_blks_read, temp_blks_written, blk_read_time, blk_write_time FROM pg_stat_statements WHERE userid in (SELECT usesysid FROM pg_user WHERE usename != any(array['postgres', 'kokumon']) and usename !~ 'test') AND mean_time > 500 AND query !~ 'INSERT INTO' AND query !~ 'CREATE'"
      master: true
      cache_seconds: 60
      metrics:
        - query:
            usage: "LABEL"
            description: "Parameterized SQL query"
        - calls:
            usage: "COUNTER"
            description: "Number of times executed"
        - total_time:
            usage: "COUNTER"
            description: "Total time spent in the statement, in milliseconds"
        - min_time:
            usage: "GAUGE"
            description: "Minimum time spent in the statement, in milliseconds"
        - max_time:
            usage: "GAUGE"
            description: "Maximum time spent in the statement, in milliseconds"
        - mean_time:
            usage: "GAUGE"
            description: "Mean time spent in the statement, in milliseconds"
        - stddev_time:
            usage: "GAUGE"
            description: "Population standard deviation of time spent in the statement, in milliseconds"
        - rows:
            usage: "COUNTER"
            description: "Total number of rows retrieved or affected by the statement"
        - shared_blks_hit:
            usage: "COUNTER"
            description: "Total number of shared block cache hits by the statement"
        - shared_blks_read:
            usage: "COUNTER"
            description: "Total number of shared blocks read by the statement"
        - shared_blks_dirtied:
            usage: "COUNTER"
            description: "Total number of shared blocks dirtied by the statement"
        - shared_blks_written:
            usage: "COUNTER"
            description: "Total number of shared blocks written by the statement"
        - local_blks_hit:
            usage: "COUNTER"
            description: "Total number of local block cache hits by the statement"
        - local_blks_read:
            usage: "COUNTER"
            description: "Total number of local blocks read by the statement"
        - local_blks_dirtied:
            usage: "COUNTER"
            description: "Total number of local blocks dirtied by the statement"
        - local_blks_written:
            usage: "COUNTER"
            description: "Total number of local blocks written by the statement"
        - temp_blks_read:
            usage: "COUNTER"
            description: "Total number of temp blocks read by the statement"
        - temp_blks_written:
            usage: "COUNTER"
            description: "Total number of temp blocks written by the statement"
        - blk_read_time:
            usage: "COUNTER"
            description: "Total time the statement spent reading blocks, in milliseconds (if track_io_timing is enabled, otherwise zero)"
        - blk_write_time:
            usage: "COUNTER"
            description: "Total time the statement spent writing blocks, in milliseconds (if track_io_timing is enabled, otherwise zero)"
```

**Comment by Red-HAP** [comment]
This dashboard is currently running in CI here: https://metrics-mnm-ci.5a9f.insights-dev.openshiftapps.com/d/nlXK6Y6Wz/koku-query-perfoamance?orgId=1

The dashboard config json has been merged into the e2e-deploy repo as templates/mnm/grafana-config/dev/dashboards/CM-Query-Performance.json

**Comment by Red-HAP** [comment]
I am going to open another ticket to get this dashboard config into prod. But it is currently blocked by the fact that the other e2e-deploy changes have not passed review yet and may be changing based on work with SRE AppInterface
